app/services/cert/certs.rb

```diff
@@ -0,0 +1,19 @@
+module Cert
+  module Certs
+    def self.ueber_cert(organization)
+      organization.debug_cert
+    end
+
+    def self.ca_cert
+      File.open(Setting[:ssl_ca_file], 'r').read
+    end
+
+    def self.ssl_client_cert
+      OpenSSL::X509::Certificate.new(File.open(SETTINGS[:katello][:ssl_client_cert], 'r').read)
+    end
+
+    def self.ssl_client_key
+      OpenSSL::PKey::RSA.new(File.open(SETTINGS[:katello][:ssl_client_key], 'r').read)
```

Comment:
For these two settings, any reason to not use the DB Setting vs the config file Settings?  

https://github.com/Katello/katello/blob/master/app/models/setting/katello.rb